.github/workflows/ci.yml

```diff
@@ -5,13 +5,13 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        ghc: ['8.4.4', '8.6.5', '8.8.4', '8.10.7', '9.0.2', '9.2.2']
+        ghc: ['8.4.4', '8.6.5', '8.8.4', '8.10.7', '9.0.2', '9.2.2', '9.4.2']
     name: Haskell GHC ${{ matrix.ghc }} cabal
     env:
       cabal_project_freeze: cabal.project.${{ matrix.ghc }}.freeze
     steps:
       - uses: actions/checkout@v2
-      - uses: haskell/actions/setup@v1.2.9
+      - uses: haskell/actions/setup@v2
```

Comment:
9.4.1 isn't supported on `haskell/actions/setup@v1.2.9` so I bumped the version.